tests/scipy_fft_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import itertools
+
+from absl.testing import absltest, parameterized
+
+from jax import test_util as jtu
+import jax.scipy.fft as jsp_fft
+import scipy.fft as osp_fft
```

Comment:
import `scipy.fftpack`, with a TODO to change to `scipy.fft` when the minimum scipy is newer.